enterprise/migrator/migrator.go

```diff
@@ -61,6 +61,7 @@ func New(mode string, jobsDB *jobsdb.HandleT, pf pathfinder.ClusterStateT) Migra
 
 func loadConfig() {
 	dbReadBatchSize = config.GetInt("Migrator.dbReadBatchSize", 10000)
+	config.RegisterDurationConfigVariable(0, &ReadHeaderTimeout, false, time.Second, []string{"ReadHeaderTimeout", "ReadHeaderTimeoutInSec"}...)
```

Comment:
shouldn't this be prefixed with `Migrator.` ?

Reply:
yeah, also `ReadHeaderTimeoutInSec` is not needed, the `Sec` suffix is deprecated cc @BonapartePC - I will fix this